examples/benchmarks/contract-reconciliation-memanto/README.md

```
@@ -0,0 +1,121 @@
+
+## Validation
+
+- ✅ All 5 queries produce deterministic golden evidence matching
+- ✅ No LLM judge required
+- ✅ No network calls, no API keys needed
+- ✅ Fully reproducible on any Python 3.10+ environment
+- ✅ 18 unit tests covering dataset, backends, and integration
+
+## Files
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)